marconi-chain-index/src/Marconi/ChainIndex/Indexers/MintBurn.hs

```diff
@@ -306,56 +306,97 @@ data instance RI.StorableQuery MintBurnHandle
   | QueryAllMintBurn (Maybe C.SlotNo)
   -- ^ Query all transactions that minted 'AssetId's until an upper bound slot in the blockchain. If
   -- the upper bound slot is 'Nothing', then we return everything.
+  | QueryBurnByAssetId C.PolicyId C.AssetName (Maybe C.SlotNo)
+  -- ^ Query all burn transactions that minted a specific 'AssetId' until an upper bound slot in the
```

Comment:
I don't think the comment is correct. Should be something like "Query all transactions that burned a specific 'AssetId' until an upper bound slot in the blockchain". Also, maybe we should say "Query all transaction events", instead of just transactions.

Reply:
Tried a better comment: 5e2f5911913a99e96c4994eed3297298da8e5074

Reply:
@eyeinsky  Needs addressing.

Reply:
Done here 8c7e260f76bac84d4ba5ab56e197c909fb069da9